src/Microsoft.DotNet.Wpf/src/PresentationCore/PresentationCore.csproj

```diff
@@ -1421,6 +1420,21 @@
     <ProjectReference Include="$(WpfSourceDir)UIAutomation\UIAutomationProvider\UIAutomationProvider.csproj" />
   </ItemGroup>
 
+  <!-- 
+    Need Microsoft.WinFx.targets to embed *.compositefont 
```

Comment:
Can PBT function before PresentationCore is built and available?

Reply:
It seems to have been fine.  I never added this change in the first place and I am not 100% sure exactly what PBT has to do with including XML files as a resource, but I wanted to preserve what was done in PresentationCore-CommonResources.csproj.  Apart from that, I definitely see the correct assemblies coming out, I can check the build logs to see what PBT is up to during PCore builds.

Reply:
Talked offline about this, and PBT doesn't need PCore just to do resource generation.